src/lib/components/cloudprogressbar/CloudProgressBar.component.js

```diff
@@ -0,0 +1,132 @@
+//@flow
+import React from 'react';
+import styled, { css, keyframes } from "styled-components";
+import * as defaultTheme from "../../style/theme";
+import type { Node } from "react";
+
+export type CloudProgressBarProps = {
+  percentage: number,
+  borderSize: string,
+  cloudColor?: string,
+  progressCloudColor?: string,
+  children: Node
+};
+
+const keyFrameCloud = props => {
+  return keyframes`
+    from {
+      height:100%
+    } 
+    to {
+      height:${100 - props.percentage}%
+    }`;
+};
+
+const Container = styled.div`
+  ${props => {
+    return css`
+      width: 100%;
+      text-align: center;
+      position: relative;
+    `;
+  }}
+`;
+
+const ContainerProgress = styled.div`
+  ${props => {
+    return css`
+      animation-duration: 1s;
+      animation-fill-mode: both;
+      animation-name: ${keyFrameCloud};
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: block;
```

Comment:
Why do we need it here? I guess could be removed.

Reply:
We need of this to align the items, and for the two clouds be aligned as well.